_docs/geyser/current-limitations.md

```diff
@@ -17,7 +17,8 @@ The following things cannot be fixed without changes to Bedrock or the Java prot
 - Various command arguments for any command that doesn't use the Minecraft Brigadier library
 - Anything that relies on tab complete or typing in the chat UI (related to the above) - Bedrock sends no packet that indicates they are in this menu
 - Unable to see banner layers past 6
-- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: https://github.com/Camotoy/GeyserHacks
+- Movement issues around bamboo due to offset differences between Java and Bedrock. The following plugin works around this, but please read the README before using: [Hurricane](/other/hurricaneh)
+- Can't move mounted entities with carrot/warped fungus on a stick(1.17.x & above). Hurricane also works around this.
```

Comment:
I'd remove the version here, and instead add e.g. camels to this list - Camels specifically are not fixed by Hurricane though.